src/internal/Modal.js

```diff
@@ -195,7 +196,7 @@ export default class Modal extends Component {
 
       if (!modalContent.hasAttribute('tabIndex')) {
         modalContent.setAttribute('tabIndex', -1);
-        warning(false, (
+        this.warningHandler(false, (
```

Comment:
That change would break https://github.com/oliviertassinari/babel-plugin-transform-dev-warning.
I have never found an elegant way to test `warning`. One their side, they use https://github.com/BerkeleyTrue/warning/blob/master/test/package/development.js#L20